.gitignore

```diff
@@ -23,3 +23,6 @@
 /docs/docs
 .yardoc
 node_modules
+
+# webpack-bundle-analyzer report
+lib/inferno/public/report.html
```

Comment:
Not a huge deal, but this doesn't seem like the right spot for a file never intended for public consumption.  Can we dump this at the project root or something like that?  Or is doing it this way just the convention?

Reply:
Moved to root